tests/unit/ConsentContinuationOut.spec.ts

```diff
@@ -198,6 +201,7 @@ describe('Consent to Continuation Out view', () => {
     // verify new Filing ID
     expect(vm.filingId).toBe(456)
 
+    jest.restoreAllMocks()
```

Comment:
Added this for `jest.spyOn` calls. Initially all newly added tests were failing and it took me some time to figure out the reason. It would be good to remember to `restoreAllMocks()` otherwise it has a trickling effect.